website/content/en/preview/upgrading/upgrade-guide.md

```diff
@@ -37,6 +37,144 @@ kubectl apply -f https://raw.githubusercontent.com/aws/karpenter{{< githubRelRef
 WHEN CREATING A NEW SECTION OF THE UPGRADE GUIDANCE FOR NEWER VERSIONS, ENSURE THAT YOU COPY THE BETA API ALERT SECTION FROM THE LAST RELEASE TO PROPERLY WARN USERS OF THE RISK OF UPGRADING WITHOUT GOING TO 0.32.x FIRST
 -->
 
+### Upgrading to `1.0.0`+
```

Comment:
We didn't mention the name change for `karpenter.sh/disruption=disrupting` to `karpenter.sh/disrupted` in here